include/seqan3/io/alignment_file/format_bam.hpp

```diff
@@ -64,12 +64,13 @@ class format_bam : private detail::format_sam_base
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    format_bam() noexcept = default; //!< Defaulted.
-    format_bam(format_bam const &) noexcept = default; //!< Defaulted.
-    format_bam & operator=(format_bam const &) noexcept = default; //!< Defaulted.
-    format_bam(format_bam &&) noexcept = default; //!< Defaulted.
-    format_bam & operator=(format_bam &&) noexcept = default; //!< Defaulted.
-    ~format_bam() noexcept = default; //!< Defaulted.
+    // string_buffer is of type std::string and has some problems with pre-C++11 ABI
```

Comment:
```suggestion
    // string_buffer is of type std::string and has some problems with pre-C++11 ABI if ctor is declared noexcept
```
? Otherwise the comment is not very helpful by itself.
You can also just delete it completely since we decided to not use noexcept explicitly anyway.